src/mailer/engine.py

```diff
@@ -165,6 +175,11 @@ def send_all():
         "django.core.mail.backends.smtp.EmailBackend"
     )
 
+    error_handler = import_string(
+        getattr(settings, 'MAILER_ERROR_HANDLER',
+            'mailer.engine.handle_backend_exception')
```

Comment:
This line needs a fix in indentation. I'd expect this layout to work:
```python
    error_handler = import_string(
        getattr(settings, 'MAILER_ERROR_HANDLER',
                'mailer.engine.handle_backend_exception')
    )
```
Not how the string is visually (or vertically) aligned with `settings`.

Error was:
> ```
> ./src/mailer/engine.py:180:13: E128 continuation line under-indented for visual indent
> ```